scpdt/tests/test_pytest_configuration.py

```diff
@@ -1,15 +1,29 @@
 import pytest
-from pathlib import PosixPath, Path
+from pathlib import PosixPath
```

Comment:
Thanks for bringing this up! You're right - using Path instead of PosixPath makes the code more platform-independent, which is definitely a better approach. 
I did a local test with Path and didn't notice any issues that may have informed my decision, so I'll go ahead and make that change while adding the second workflow.